lib/px/utils/pxpayload.lua

```diff
@@ -206,6 +206,15 @@ function PXPayload:validate(cookie)
 end
 
 function PXPayload:is_sensitive_route()
+    if self.px_config.sensitive_routes ~= nil then
+        for i, p in ipairs(self.px_config.sensitive_routes) do
```

Comment:
do you have to go over pairs if you only care for the value?

Reply:
yeah, LUA doesn't have a search() function for tables.